mpdel-core.el

```diff
@@ -81,12 +81,14 @@ list with this entity.  Otherwise, return nil."
 (defun mpdel-core-replace-current-playlist ()
   "Replace current playlist with selected entities."
   (interactive)
-  (libmpdel-current-playlist-replace (mpdel-core-selected-entities)))
+  (when (y-or-n-p "Really replace current playlist? ")
```

Comment:
Maybe `yes-or-no-p` is more appropriate and those that set it to have the same definition as `y-or-n-p` won't be any wiser?

Reply:
I have no preference. The [manual](https://www.gnu.org/software/emacs/manual/html_node/elisp/Yes_002dor_002dNo-Queries.html#index-yes_002dor_002dno_002dp) says that:

> ‘yes-or-no-p’ requires more work from the user than ‘y-or-n-p’ and
> is appropriate for more crucial decisions.